client/components/backup-retention-management/retention-options/retention-option-card.tsx

```diff
@@ -0,0 +1,75 @@
+import classnames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
+import type { TranslateResult } from 'i18n-calypso';
+
+interface RetentionOptionCardProps {
+	label: string;
+	spaceNeeded: TranslateResult | string;
```

Comment:
Won't the `TranslateResult` include `string` already?

Also, it's an odd prop name - it sounds like a boolean but it's not. Thoughts?

Reply:
> Won't the TranslateResult include string already?

It seems [it is actually defined as a React.ReactChild](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-calypso/types/index.d.ts#L59) and Typescript doesn't like the assumption that it is a string.

Either way, we revisited it and noticed that it is preferable to pass the space needed in bytes, so we made a little refactor there.

Addressed in #73303